Author: Andreas Tille <tille@debian.org>
Last-Update: Sun, 18 Dec 2016 08:31:42 +0100
Description: Ignore cwltest for the moment
 See the hint in
   https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=845982#62

--- cwltool.orig/setup.py
+++ cwltool/setup.py
@@ -49,7 +49,6 @@
           'shellescape >= 3.4.1, < 3.5',
           'schema-salad >= 2.2.20170111180227, < 3',
           'typing >= 3.5.2, < 3.6',
-          'cwltest >= 1.0.20161227194859'
       ],
       test_suite='tests',
       tests_require=[],
